starky2/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+#![allow(incomplete_features)]
+#![allow(clippy::too_many_arguments)]
+#![allow(clippy::type_complexity)]
+#![feature(generic_const_exprs)]
+
+pub mod config;
+pub mod constraint_consumer;
+pub mod cross_table_lookup;
+mod get_challenges;
+// pub mod mock_stark;
```

Comment:
Is this a work in progress?

Reply:
It's a relic from the beginning of the branch, when I wanted to do things generically. I've removed it.
It's going to be a bit tricky to do testing with the non-generic approach. Like right now I've just added dummy columns to the CPU and Keccak STARKs, but once we add real columns and constraints, the test will fail if just one of the table still has TODOs.

Reply:
Makes sense, we can just test with the real thing. I think we can use TODO comments (with no code) rather than `todo!()` etc. so that tests will generally be green, unless we add constraints before witness generation or what not.